libnavigation-ui/src/main/java/com/mapbox/navigation/ui/route/MapRouteLine.kt

```diff
@@ -278,6 +294,8 @@ internal class MapRouteLine(
             style.getLayer(PRIMARY_ROUTE_LAYER_ID)?.setProperties(lineGradient(expression))
             hideShieldLineAtOffset(vanishPointOffset)
         }
+
+        routeLineInitializedCallback?.onInitialized(PRIMARY_ROUTE_LAYER_ID)
```

Comment:
The alternative route layer is rendered below the primary layer, so if the developer tried positioning a custom layer right below the primary route, it will be rendered in between primary and alternative route layers. To cover all use-cases, we possibly should pass layer IDs of both primary and alternative route layers here.